requirements/requirements.txt

```diff
@@ -7,12 +7,12 @@ padacioso~=0.2, >=0.2.1
 adapt-parser>=1.0.0, <2.0.0
 
 ovos-utils>=0.0.38
-ovos_bus_client<0.1.0, >=0.0.8
+ovos_bus_client<0.1.0, >=0.0.9a15
```

Comment:
If I'm following correctly, 0.0.9a2 introduced `ovos-utils 0.1.0aX` dependencies, so ovos-core 0.0.8 can probably only require ovos-bus-client 0.0.8 in order to support ovos-utils < 0.1.0